projects/novo-examples/src/updates/v10.md

```diff
@@ -0,0 +1,48 @@
+---
+section: updates
+page: v10-announce
+title: v10
+order: 5
+tag: new
+---
+
+🎉 April 2024 (version 10)
+===========================
+
+**Announcement**: Novo Elements is now on Angular 17!
+
+This brings Novo Elements up to the latest versions of both Angular and Typescript. We have also removed the long-since deprecated Table component and Dragula add-on. 
+
+## Upgrading to v10
+- You may need to update your node version as Angular 17 requires at least `^18.13.0` or `^20.9.0`.
+- If your application is using the Novo Elements `dragula` add-on, you will need to refactor it using the guide below.
+- If your application is still using the Novo Elements `table` component, you will need to refactor it to use the `novo-data-table` component.
+
+## Dragula
+There are a few options for Novo Elements users who wish to use drag and drop functionality on our components.
+
+For basic new implementations we recommend our new lightweight [drag-drop component](https://bullhorn.github.io/novo-elements/docs/#/utils/drag%20and%20drop) which should handle most simple use cases going forward.
+
+If you have more advanced needs or are refactoring a component that is currently using the previous Novo Elements dragula add-on directive and service, we recommend switching over to use the [ng2-dragula](https://github.com/valor-software/ng2-dragula) library. This was forked from the same library we originally forked @bullhorn/dragula from, but has been receiving consistent updates for the past 6 years (while ours hasn't), and has its own directive and service which function almost identically to our Novo Elements add-on directive and service which were just removed. Some key differences if you're going this route:
```

Comment:
If you have more advanced needs or are refactoring a component that is currently using the previous Novo Elements dragula add-on directive and service, we recommend switching over to using the [ng2-dragula](https://github.com/valor-software/ng2-dragula) library. This library is better maintained. It has a very similar API which greatly reduces the refactor effort. If you are choosing to leverage this library here are some key differences to keep in mind: